pkg/apis/experimental/v1alpha1/custom_cluster_builder_validation_test.go

```diff
@@ -23,10 +23,8 @@ func testCustomBuilderValidation(t *testing.T, when spec.G, it spec.S) {
 		},
 		Spec: CustomClusterBuilderSpec{
 			CustomBuilderSpec: CustomBuilderSpec{
-				Tag: "some-registry.io/custom-builder",
-				Stack: Stack{
-					BaseBuilderImage: "some-registry.io/base-builder",
-				},
+				Tag:   "some-registry.io/custom-builder",
+				Stack: "some-stack-ref",
 				Store: "some-registry.io/store",
```

Comment:
This is not related to this PR but we should consider renaming some of these test fields to read to refs and not images in a registry to avoid confusion.